rubocop.yml

```
@@ -542,6 +542,9 @@ Layout/DefEndAlignment:
   - start_of_line
   - def
 
+Layout/BeginEndAlignment:
+  EnforcedStyleAlignWith: start_of_line
```

This is the default config, so we can just enable the cop.

```suggestion
  Enabled: true
```

---

Ok.  I wasn't sure if we wanted to rely on the default not changing.